Frontend/CO.CDP.OrganisationApp/Session.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Text.Json;
+
+namespace CO.CDP.OrganisationApp
+{
+    public class Session(IHttpContextAccessor httpContextAccessor) : ISession
```

Comment:
All methods require the httpContextAccessor, it makes sense to set it in the constructor.

` private readonly IHttpContextAccessor _httpContextAccessor;

        public Session(IHttpContextAccessor httpContextAccessor)
        {
            _httpContextAccessor = httpContextAccessor;
        }`

Reply:
It is set in the constructor @glgoaco  
C# 12 introduces [primary constructors](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/instance-constructors#primary-constructors), a concise syntax to declare constructors whose parameters are available anywhere in the body of the type.